--- installwatch-0.6.3/Makefile~	Sun Dec 23 00:37:06 2001
+++ installwatch-0.6.3/Makefile	Sat Feb 23 23:14:50 2002
@@ -25,10 +25,9 @@
 	./create-localdecls
 
 install: all
-	if [ -r $(LIBDIR)/installwatch.so ]; then rm  $(LIBDIR)/installwatch.so; export LD_PRELOAD=""; cp installwatch.so $(LIBDIR); LD_PRELOAD=$(LIBDIR)/installwatch.so; else cp installwatch.so $(LIBDIR); fi
+	if [ -r $(DESTDIR)$(LIBDIR)/installwatch.so ]; then rm  $(DESTDIR)$(LIBDIR)/installwatch.so; export LD_PRELOAD=""; cp installwatch.so $(DESTDIR)$(LIBDIR); LD_PRELOAD=$(LIBDIR)/installwatch.so; else cp installwatch.so $(DESTDIR)$(LIBDIR); fi
 	
-	sed -e "s|#PREFIX#|$(PREFIX)|" < installwatch > $(BINDIR)/installwatch
-	chmod 755 $(BINDIR)/installwatch
+	sed -e "s|#PREFIX#|$(PREFIX)|" < installwatch > $(DESTDIR)$(BINDIR)/installwatch
 
 uninstall:
 	rm $(LIBDIR)/installwatch.so
